server/modules/index.js

```diff
@@ -2,91 +2,91 @@ module.exports = [
     {
         name: 'util.tokenGenerator',
         dependencies: [],
-        require: __dirname + '/token-generator'
+        module: require(__dirname + '/token-generator')
```

Comment:
The reason why I proposed this change is related to `es module` loading 
(Not sure if its good idea to use it in node). If we do it supports some optimization, but require a static definition of `require / import`.

I m not sure why U mean by lazy I think all modules has been required while iterating over `modules` in `container.js`

Node loads modules always single time (no matter how many `imports  / require` do you have - it will be loaded only once (result of `module.export is saved in memory`)